script.js

```diff
@@ -1,4 +1,199 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+import { DECK } from "./deck.js";
+let cardDeck = DECK;
+const GAMESTATES = ["draw-cards", "stand-or-hit", "show-results"];
+let gameState = GAMESTATES[0];
+let results = {
+  dealer: {
+    score: 0,
+    cardsDrawn: [],
+    cardNames: [],
+    cardTotal: 0,
+  },
+  "Player 1": {
+    score: 0,
+    cardsDrawn: [],
+    cardNames: [],
+    cardTotal: 0,
+  },
+};
+const renderButtons = () => {
+  document
+    .querySelectorAll(".btn--conditional")
+    .forEach((e) => (e.style.display = "inline-block"));
+};
+const hideButtons = () => {
+  document
+    .querySelectorAll(".btn--conditional")
+    .forEach((e) => (e.style.display = "none"));
+};
+const shuffleArray = (array) => {
+  for (let i = array.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1)); // FisherYates Algorithm
+    const temp = array[i];
+    array[i] = array[j];
+    array[j] = temp;
+  }
+};
+const drawCards = (numCards, player) => {
+  for (let _ = 0; _ < numCards; _++) {
```

Comment:
Check out this syntax and see if it removes the need for you to use underscores: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of